wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/DeferredCommand.java

```diff
@@ -28,9 +28,10 @@ public class DeferredCommand extends Command {
   private Command m_command = m_nullCommand;
 
   /**
-   * Creates a new DeferredCommand that runs the supplied command when initialized, and ends when it
-   * ends. Useful for lazily creating commands at runtime. The {@link Supplier} will be called each
-   * time this command is initialized. The Supplier <i>must</i> create a new Command each call.
+   * Creates a new DeferredCommand that runs the supplied command when initialized without scheduling 
```

Comment:
I'm not a fan of the "without scheduling it" phrasing, I think "directly" would be more unambiguous?